src/main.js

```diff
@@ -55,22 +59,30 @@ if (crashReporterDirectory) {
 		}
 	}
 
-	// Crashes are stored in the temp directory by default, so we
+	// Crashes are stored in the crashDumps directory by default, so we
 	// need to change that directory to the provided one
 	console.log(`Found --crash-reporter-directory argument. Setting temp directory to be '${crashReporterDirectory}'`);
```

Comment:
@deepak1556 the console log message talks about `temp`, but it looks like this is `crashDumps` directory now?